### ISSUE TITLE
Missing Support for Platform Script

### ISSUE THREAD

**Issue by BestNoob** [ManiaControl/ManiaControl#231]
https://forum.maniaplanet.com/viewtopic.php?f=659&t=45303
https://forum.maniaplanet.com/viewtopic.php?f=538&t=45468

**Comment by kremsy** [comment]
What is missing?

**Comment by axelalex2** [comment]
I will close this off, if nothing is added here.

**Comment by BestNoob** [comment]
> 
> 
> What is missing?

Everything xD 
Have u ever take a look into this Title/script ?

**Comment by axelalex2** [comment]
Quite certainly, no one of us did. And our main focus doesn't necessarily lie on gamemodes, which are rarely played online. ManiaControl needs other updates first, especially usability features for coders and players, so if you want us to support platform mode, you need to help us out a bit.

**Comment by BestNoob** [comment]
Yeah but .... i tested all the Servercontrollers and only UASECO makes me happy.
But specially for the mod Servers i want to use other server controllers like expansion/maniacontrol both also supported Platform in the past ....... so i hoped for those but ...
